tests/system/Session/Handlers/Database/RedisHandlerTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Session\Handlers\Database;
+
+use CodeIgniter\Session\Handlers\RedisHandler;
+use CodeIgniter\Test\CIUnitTestCase;
+use Config\App as AppConfig;
+use Redis;
+
+/**
+ * @internal
+ */
+final class RedisHandlerTest extends CIUnitTestCase
+{
+    private string $sessionName     = 'ci_session';
+    private string $sessionSavePath = 'tcp://127.0.0.1:6379';
+    private string $userIpAddress   = '127.0.0.1';
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        if (! class_exists(Redis::class)) {
```

Comment:
How about moving this to setUpBeforeClass? Or, using a class docblock instead, like `@requires extension redis`?

Reply:
Oh, thanks. It is better.
Done.